### PR TITLE
:bar_chart: conflict: Three military expenditure datasets

### PR DESCRIPTION
**Pull request by paarriagadap**
Main issue: https://github.com/owid/owid-issues/issues/164

Originally, the idea was to add several datasets and merge them to construct a military expenditure OMM, but we scrapped that.

Instead, now we have three different dataset in the same PR:

- [SIPRI Military Expenditure Database](http://staging-site-update-sipri-military-expenditure/admin/datasets/6610)
- [Global Military Spending Dataset](http://staging-site-update-sipri-military-expenditure/admin/datasets/6611)
- [National Material Capabilities](http://staging-site-update-sipri-military-expenditure/admin/datasets/6634)

### CODE REVIEW

File: etl/steps/data/garden/sipri/2024-07-08/military_expenditure.meta.yml
Comment (veronikasamborska1994) [inline etl/steps/data/garden/sipri/2024-07-08/military_expenditure.meta.yml:9]: I'd move the "Civil defense and current expenditures for previous
military activities are excluded." to the footnote to shorten the subtitle a bit and also so it's consistent with the other chart Screenshot 2024-07-30 at 11 01 52

**Review by veronikasamborska1994 — APPROVED** [review]
Screenshot 2024-07-30 at 10 55 37
LGTM! I am not sure if people will actually read such long and technical descriptions from producer but I guess there isn't much you can do about it. one small thing - the source on this chart still says "Our World in Data based on..." I am pretty sure we have stopped doing this but maybe I am misremembering something

File: etl/steps/data/garden/cow/2024-07-26/national_material_capabilities.py
Comment (paarriagadap) [reply etl/steps/data/garden/cow/2024-07-26/national_material_capabilities.py:54]: In this case, it's because the countries are using a notation implemented by the COW Project: I am translating the codes to country names standardized in that other dataset. I don't mind about keeping (very) old entities: they are only circumscribed to their period.

**Comment by paarriagadap** [comment]
Thank you @veronikasamborska1994! The _OWID based on..._ chart will be deleted and redirected, but I can't do that before merging because that chart is referenced on the webpage.

**Pull request merged**
